.github/DOCUMENTATION_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+# How to contribute to MFTF docs
+
+MFTF documentation is kept within the /docs/ folder in this repository.
+We welcome contributions to the documentation.
+This page describes the process for submitting documentation and serves as a template for a properly written content.
+
+The contribution workflow for the Magento functional testing framework (MFTF) documentation is the same as submitting code.
+
+1. Create a branch from the `develop` branch in the [MFTF repo][].
+1. Make edits/additions/deletions as needed.
+1. Submit your PR to the `develop` branch.
+
+Once submitted, a member of the documentation team will review and merge it.
+We will inform you if it needs any additional processing.
+
+The documentation in this repository is used as the source for the [MFTF documentation][].
+Any changes to the Table of Contents will need to be made via a separate PR in the regular [devdocs repo][].
```

Comment:
1. As a developer who is contributing to this repository, how do I know what Table of Contents is this sentence about? There is no Table of Contents here.

2. Change "repo" to "repository". Avoid abbreviations.

Reply:
As a code contributor, I do not know what is devdocs.